css/properties/grid-gap.json

```diff
@@ -0,0 +1,41 @@
+{
+  "css": {
+    "properties": {
+      "grid-gap": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-align-3/#gap-legacy",
+          "support": {
+            "chrome": {
+              "version_added": "≤80"
```

Comment:
Only testing [in grid and flex](https://codepen.io/bsmth/pen/PoVPqVa), but for Chrome, the legacy syntax works for flexbox in 84, grid as far back as 57 in browserstack.